### PR TITLE
Review survey form project

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
setup linters 
add descriptive readMe file
add past project 
fix linter errors

### CODE REVIEW

**Review by Kandy-Peter — COMMENTED** [review]
Hi @Owusu-Desmond ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request** 🔴 🔴 ⛔
You have submitted a project, that has no review.. please make sure you add some review inline comments and changes to be made. you are almost there 😄

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Invalid Code Review Request does not count into the code reviews limit.

**Comment by Owusu-Desmond** [comment]
Hi @Kandy-Peter 

Please can you further explain the review that you are expecting in the project ? or have I submitted unsuspected project.

**Comment by Kandy-Peter** [comment]
Hi @Owusu-Desmond,
As required, you have to perform a review on this project and apply HTML & CSS best practices introduced by the [How to give a code review?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) lesson. Once you are done, submit for review and the next CR will give you a feedback on how you performed your review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+</head>
+<body>
+    <div class="container">
+      <div class="header">
+        <h1 class="main-header" id="title">HTML AND CSS SURVEY FORM</h1>
+        <p id="description">Fill the form with your correct detail to apply to be a student at microverse.</p>
+    </div>
+      <form id="survey-form">
+        <div class="form-group">
+          <label id="name-label" for="name">Name</label>
```
Comment (Owusu-Desmond) [inline index.html:18]: Please you need to use html semantic element like **main** tag to wrap the form section

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          <input type="text" name="name" id="name" class="form-control" placeholder="Enter your name" required />
+        </div>
+        <div class="form-group">
+          <label id="email-label" for="email">Email</label>
+          <input type="email" name="email" id="email" class="form-control" placeholder="Enter your email address" required />
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:24]: Please you need consider html semantic element `section` to wrap this form group as container

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="form-group">
+          <label id="number-label" for="number">Age</label>
+          <input min="18" max="50" type="number" name="number" id="number" class="form-control" placeholder="Enter your age" required />
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:28]: Please you need consider html semantic element `section` to wrap this form group as container

File: index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="form-group">
+          <p class="form-group-description">Select your educational level</p>
+          <select name="level" id="dropdown">
+            <option label="PSL" value="Primary School Level"></option>
+            <option label="SSL" value="Secondary School Level"></option>
+            <option label="UL" value="University Level"></option>
+          </select>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:36]: Please you need consider html semantic elements like `section` tag to wrap this form group as container

File: index.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            </label>
+          <label>
+            <input name="user-recommend" value="definitely" type="radio" class="input-radio"
+                checked />Not Sure
+          </label>
+          <label>
+            <input name="user-recommend" value="definitely" type="radio" class="input-radio"
+                checked />Definitely No
+          </label>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:51]: Please you need consider html semantic element like `section` tag to wrap this form group as container

File: index.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+            </p>
+            <select id="most-like" name="mostLike" class="form-control" required>
+                <option disabled selected value>Select an option</option>
+                <option value="challenges">JavaScript</option>
+                <option value="projects">PHP</option>
+                <option value="community">GO</option>
+                <option value="openSource">PYTHON</option>
+                <option value="openSource">Others</option>
+            </select>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:64]: Please you need consider html semantic element like `section` tag to wrap this form group as container

File: index.html [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+            <label><input name="prefer" value="front-end-projects" type="checkbox" class="input-checkbox" />
+                Numbers of classes</label>
+            <label>
+                <input name="prefer" value="back-end-projects" type="checkbox" class="input-checkbox" />
+                Numbers of Tutors </label>
+            <label><input name="prefer" value="data-visualization" type="checkbox" class="input-checkbox" />
+                Education Curriculum</label>
+            <label><input name="prefer" value="challenges" type="checkbox" class="input-checkbox" />Land and Labour</label>
+        
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:81]: Please you need consider html semantic element like `section` tag to wrap this form group as container

File: index.html [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="form-group">
+          <p class="form-group-description">Write an essay here telling as the reason why you want to join microverse.</p>
+          <textarea name="essay-area" id="assay-area" cols="60" rows="5"></textarea>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:85]: Please you need consider html semantic element like `section` tag to wrap this form group as container

File: index.html [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="form-group">
+          <input type="submit" id="submit" value="Submit">
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:88]: Please you need consider html semantic element like `section` tag to wrap this form group as container

File: index.html [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML AND CSS SURVEY FORM</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <div class="container">
+      <div class="header">
+        <h1 class="main-header" id="title">HTML AND CSS SURVEY FORM</h1>
+        <p id="description">Fill the form with your correct detail to apply to be a student at microverse.</p>
+    </div>
```
Comment (Owusu-Desmond) [inline index.html:15]: Please consider using the html header tag to wrap the header part of your application.

**Review by Owusu-Desmond — COMMENTED** [review]
Please try to use html semantic tags like `header`, `main`, `section` ,  for each section in your project

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please, ensure that the Pull Request you opened utilises the "Review changes" feature from GitHub instead of just competing the code. This helps users to know the PR state(Approved or requested changes). You can this short [Video](https://youtu.be/lSnbOtw4izI) for more understanding.
- [ ] I would also suggest you use summarised content as a message to the user who receives the review. You can pick an idea from mine(the look and the logic but not the content).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 6 above)
Comment (Owusu-Desmond) [reply index.html:64]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

File: index.html (hunk 5 above)
Comment (Owusu-Desmond) [reply index.html:51]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

File: index.html (hunk 7 above)
Comment (Owusu-Desmond) [reply index.html:81]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

File: index.html (hunk 8 above)
Comment (Owusu-Desmond) [reply index.html:85]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

File: index.html (hunk 9 above)
Comment (Owusu-Desmond) [reply index.html:88]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

File: index.html (hunk 10 above)
Comment (Owusu-Desmond) [reply index.html:15]: Consider using `h2`-`h6` elements to add identifying headings to all sections, or else use a `div` element instead for any cases where no heading is needed

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Owusu-Desmond 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required changes  :recycle::recycle: 
### - As requested by the previous reviewer, I think it would be better if you add a good summary after you completed the review by **adding gifs** to make it more friendly, **highlighting** some efforts improved by the student, etc. You can use one template of your previous received reviews or the current one as an example. Just edit your summary and add the performed one.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — COMMENTED** [review]
Hi @Owusu-Desmond , 

Congratulation's, your pull request have successfully  approved , now what next you have to do is continue to merge it.

[congratulation](https://www.bing.com/th?id=OIP.VRVCXuykH1ABUvdPP3bt9wHaDf&w=308&h=145&c=8&rs=1&qlt=90&o=6&pid=3.1&rm=2)

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Owusu-Desmond 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**